metrics/pom.xml

```diff
@@ -56,12 +70,20 @@
             <groupId>com.typesafe</groupId>
             <artifactId>config</artifactId>
         </dependency>
-
-        <!--Test-->
         <dependency>
-            <groupId>org.hamcrest</groupId>
-            <artifactId>hamcrest</artifactId>
-            <scope>test</scope>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>${json.version}</version>
+        </dependency>
```

Comment:
I'm not familiar with this `org.json` library, but looking at the code it looks like it duplicates what we are already doing with Jackson. If that's the case I would ask that you use Jackson instead as I would like to avoid a needless proliferation of dependencies.

Reply:
The json returned does not have perfect json elements - datapoint. Jackson was tried but did not work. This library needs to be used for processing.

Reply:
I need help understanding what won't work.